bin/index.js

```diff
@@ -59,6 +58,51 @@ function simplifySchema(schema) {
   }, {});
 }
 
+async function getSchemaFromAirtable(settings) {
+  if (settings.mode === 'manual') {
+    console.log(
+      '\n## Getting Schema using Manual mode ##\nWill look for `schemaRaw.json` in specified input folder...\n\n'
+    );
+    const schema = readSchemaFromFile(settings);
+    const baseUrl = `https://airtable.com/login?continue=/${settings.baseId}/api/docs`;
+
+    if (!schema) {
+      throw `No Schema Found: Please create a schemaRaw.json file in the input folder.\n\n Navigate to this page in the browser: ${baseUrl} \n\nRun this script in the console:\n\n${script}\n\nThis will save the output to your clipboard. Open up schemaRaw.json and paste!`;
+    } else {
+      console.log('Found Schema file in `schemaRaw.json` file');
+      console.log(
+        '\n*** Make sure this is the latest schema ***\nIf you need to update the schema follow these instructions:\n\n'
+      );
+      console.log(
+        `1. Navigate to this page in the browser: ${baseUrl} \n2. Run this script in the console:\n\n${script}\n\nThis will save the output to your clipboard.\n3. Open up schemaRaw.json, delete everything, and paste!\n\n`
+      );
+      return schema;
+    }
+  } else {
+    const mode = settings.mode === 'auto-headless' ? settings.mode : 'auto';
+    console.log(
+      `\n## Getting Schema using ${mode} mode. Will fetch schema from airtable.com... ##\n\n${
+        mode === 'auto'
+          ? 'Launching browser...'
+          : 'Launching headless browser...'
+      }`
+    );
+
+    // Load config vars
+    require('dotenv-safe').config({
+      path: '.airtable.env',
+      example: path.resolve(__dirname, '../.env.example')
```

Comment:
Same comment from #15 - if we're going to call this `.airtable.env`, any thoughts on having the example file be called `.example.env` (or vice versa) ?

Reply:
fixed in future PR